standard/structs.md

```diff
@@ -138,6 +140,56 @@ struct_member_declaration
 
 Except for the differences noted in [§16.4](structs.md#164-class-and-struct-differences), the descriptions of class members provided in [§15.3](classes.md#153-class-members) through [§15.12](classes.md#1512-static-constructors) apply to struct members as well.
 
+### §struct_readonly Readonly members
+
+An instance member definition or property accessor of an instance property that includes the `readonly` modifier has the following restrictions:
```

Comment:
```suggestion
An instance member definition or accessor of an instance property, indexer, or event that includes the `readonly` modifier has the following restrictions:
```

Reply:
Then below, it says "the member" even if at this point it had been "the accessor." Does the spec have a handy concept of "function member body" or something like that?